source/materials/MaterialsList.cc

```diff
@@ -15,55 +15,85 @@
 #include <G4Element.hh>
 #include <G4NistManager.hh>
 
+#include <vector>
+#include <utility> 
+
 using namespace nexus;
 using namespace CLHEP;
 
-
 namespace materials {
-  G4Material* GXe(G4double pressure, G4double temperature)
-  {
-    G4Material* mat = GXe_bydensity(GXeDensity(pressure),
-                                    temperature, pressure);
+    // Calculation of Compressibilty factor from https://next.ific.uv.es/DocDB/0011/001183/001/GasDensity_2021-04-23.pdf
+    G4double CompressibilityFactor(G4double pressure, G4double temperature) {
```

Comment:
If I understand correctly, it's this formula the one that needs the pressure in bar and the temperature in kelvin, is that right? In that case, I would make clear that the code transform whatever T/P is passed by the user in those units, like this:

`double T_r = (temperature/kelvin)/289.733;`
`double P_r = (pressure/bar)/58.420;`

I think that this way it's clearer.